controllers/datadogagent/component/clusteragent/default.go

```diff
@@ -115,6 +115,12 @@ func defaultPodSpec(dda metav1.Object, volumes []corev1.Volume, volumeMounts []c
 		// 	RunAsNonRoot: apiutils.NewBoolPointer(true),
 		// },
 	}
+
+	// set container
+	container := &podSpec.Containers[0]
+	container.LivenessProbe = apicommon.GetDefaultLivenessProbe()
```

Comment:
Any reason not to inline this similar to Env, VolumeMounts, etc above?

Reply:
No reason! I''ll adjust and test again.